views.go

```diff
@@ -21,6 +24,27 @@ import (
 
 const maxLogBytes = 4 * 1024 * 1024 // 4 MB
 
+const (
+	corsOriginsEnvVariable = "LK_CORS_ORIGINS"
+)
+
+func getCorsOrigins() []string {
```

Comment:
maybe do we want to do this in an init() function and compile the regexes in it?

Reply:
Is there any benefit to doing that vs what we currently do? We use this method on evergreen